Enhance Daikin integration documentation (#41471)
home-assistant/home-assistant.github.io: source/_integrations/daikin.markdown

If your unit is not in the list above there is another option, to buy and install an [ESP32-Faikin](https://github.com/revk/ESP32-Faikin).

{% include integrations/config_flow.md %}

> If your device is set up with password, use that, if it has a API-key, use that, in all other cases leave the fields blank.

Reviewer: ```suggestion
If your device is set up with password, use the password. If it has an API key, use the API key. In all other cases, leave the fields blank.
```
